From 30f0dc10ce3e41451c6429ee4c79b3ba09a391a1 Mon Sep 17 00:00:00 2001
From: Zhangfei Gao <zhangfei.gao@linaro.org>
Date: Tue, 08 Aug 2017 12:44:47 +0800
Subject: [PATCH] common: fix ansible location

When deploying on arm64, common reports error:
stat /usr/bin/ansible: no such file or directory

The reason is ansible is installed to /usr/local/bin/ansible
rather that /usr/bin/ansible
Fix the issue to call ansible directly, to consider both cases.

Change-Id: Ie90bd9c4d7d3a4b7b0aef884351cb58ed13ebd11
Signed-off-by: Kevin Zhao <kevin.zhao@linaro.org>
Signed-off-by: Zhangfei Gao <zhangfei.gao@linaro.org>
---

diff --git a/ansible/roles/common/handlers/main.yml b/ansible/roles/common/handlers/main.yml
index 8f5ef79..9aef59e 100644
--- a/ansible/roles/common/handlers/main.yml
+++ b/ansible/roles/common/handlers/main.yml
@@ -45,7 +45,7 @@
     - Initializing toolbox container using normal user
 
 - name: Initializing toolbox container using normal user
-  command: docker exec -t kolla_toolbox /usr/bin/ansible --version
+  command: docker exec -t kolla_toolbox ansible --version
   changed_when: false
 
 - name: Restart cron container
